dagster_university/__init__.py

```diff
@@ -28,4 +30,38 @@
     jobs=all_jobs,
     schedules=all_schedules,
     sensors=all_sensors,
+    metadata={
+        "lessons": ["1","2","5","7","8","9"]
+    },
 )
+
+# defs = Definitions(
```

Comment:
Could also put all these different Definitions behind an env variable or something. Trying to think of how to make this more elegant.